test/ProductConstructionService.ScenarioTests/ScenarioTestBase.cs

```diff
@@ -281,6 +281,63 @@ protected async Task CheckNonBatchedGitHubPullRequest(
     protected static string GetExpectedCodeFlowDependencyVersionEntry(string sourceRepo, string targetRepoName, string sha, int buildId) =>
         $"<Source Uri=\"{GetGitHubRepoUrl(sourceRepo)}\" Mapping=\"{targetRepoName}\" Sha=\"{sha}\" BarId=\"{buildId}\" />";
 
+    protected async Task CheckGitHubPullRequestWithTargetDirectories(
```

Comment:
Why are these methods in the base classes when they are only used in one test?

Can we just put them in the child class?